main/java/TestForCartClass.java

```diff
@@ -0,0 +1,12 @@
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+
+public class TestForCartClass extends BeforeAfter {
+
+    @Test
+    void scenarioGetCartName() {
+        assertEquals("new-cart", newCart.getCartName());
```

Comment:
Not really good test. Add test for total price

Reply:
Test is replaced with test for Total Price.